packages/docusaurus/src/webpack/utils.ts

```diff
@@ -122,6 +123,7 @@ export function getBabelOptions({
       babelrc: false,
       configFile: babelOptions,
       caller: {name: isServer ? 'server' : 'client'},
+      exclude: new RegExp(`"/${STATIC_DIR_NAME}/"`), // exclude static dir
```

Comment:
is it normal to have " around the path?

won't this kick in for any /static/ folder (what if users have src/pages/static/xyz for example?)

What about the else case (babelOptions object?)